src/net/http/tailscale.go

```diff
@@ -0,0 +1,13 @@
+package http
+
+var roundTripEnforcer func(*Request) error
+
+func SetRoundTripEnforcer(f func(*Request) error) {
+	if f == nil {
+		panic("nil func")
+	}
+	if roundTripEnforcer != nil {
```

Comment:
It's useful to be able to set this back to nil after a test runs. I've run into that issue for the other hooks that we have.

Reply:
It's meant to be installed exactly once in init. If we want per-test control, we provide that ourselves in our installed hook.

The other two hooks are like this too.